tools/FatesPFTIndexSwapper.py

```diff
@@ -144,15 +145,15 @@ def main(argv):
 
     fp_in  = netcdf.netcdf_file(input_fname, 'r')
 
-    for key, value in sorted(fp_in.dimensions.iteritems()):
+    for key, value in sorted(fp_in.dimensions.items()):
```

Comment:
is this a python 2 vs python 3 thing?

Reply:
I should say that my python brain is stuck firmly in the aughts as regards to python versioning.  if we are making the switch to python 3, that's great and long overdue, but just let me know that that is a line that we've crossed.

Reply:
Sorry, yes this should have been a separate issue. I think python 3 only uses items now.

Reply:
great, let's make the change. thanks.

Reply:
thanks, yeah if we make this py2/3 compliant that is win win